samples/iolib/src/main/cpp/player/SampleBuffer.cpp

```diff
@@ -74,20 +74,20 @@ void resampleData(const ResampleBlock& input, ResampleBlock* output, int numChan
     float *outputBuffer = new float[numOutFrames];    // multi-channel buffer to be filled
     output->mBuffer = outputBuffer;
 
-    int numOutputFrames = 0;
-    int inputFramesLeft = input.mNumFrames;
-    while (inputFramesLeft > 0) {
+    int numOutputSamples = 0;
+    int inputSamplesLeft = input.mNumSamples;
+    while (inputSamplesLeft > 0) {
```

Comment:
Adding 8 is not enough. The sinc based resamplers can have unpredictable latency.

Also condition on:

    && numOutputSamples  < numOutAllocated

Reply:
I changed to 200. Let me know if you want a bigger number

Reply:
Changed back to 8. We can consider increasing this number later. Thanks!